<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
						background-color: black;
						}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="login_button">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
					color: white;}</string>
    </property>
    <property name="text">
     <string>ZALOGUJ!</string>
    </property>
   </widget>
   <widget class="QLabel" name="hello_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>391</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
			color: white;
			}</string>
    </property>
    <property name="text">
     <string>Zaloguj się</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="username">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
					background-color: black;
					border-radius: 20px solid white;
					color: white;
				}</string>
    </property>
    <property name="placeholderText">
     <string>Podaj swoje imię:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_counter">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>688</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
